docroot/sites/blt.development.services.yml

```diff
@@ -3,6 +3,6 @@
 # Enable twig debugging.
 parameters:
   twig.config:
-    debug: true
+    debug: false
```

Comment:
I think this should stay set to `true` as it is only loaded in the local development environment.